js/app.js

```diff
@@ -140,7 +141,9 @@
    * to set the context to the correct object (the current context)
    *   and passing two aditional arguments, event and 'light-green'
    */
-
+   btn8.addEventListener('click', function(event){
+    setMyLightClass.call(btn8, event, 'light-green');
+   });
```

Comment:
This one is asking for the current context. Replace 'btn8' with the current context